### PR TITLE
Update arc extension

### PR DESCRIPTION
**Pull request by Visual-Studio-Coder**
## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/arc/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Arc Changelog
 
+## [Improvements] - 2023-09-11
```
Comment (danulqua) [inline extensions/arc/CHANGELOG.md:3]:
```suggestion
## [Improvements] - {PR_MERGE_DATE}
```

File: extensions/arc/package.json [hunk 2]
```diff
@@ -10,7 +10,18 @@
   "categories": [
     "Web",
     "Applications",
-    "Productivity"
+    "Productivity",
+    "Web"
```
Comment (danulqua) [inline extensions/arc/package.json:14]:
```suggestion
    "Productivity"
```
Duplicated "Web" category

File: extensions/arc/src/arc.ts [hunk 3]
```diff
@@ -148,6 +148,23 @@ export async function makeNewWindow(options: MakeNewWindowOptions = {}): Promise
     end tell
   `);
 }
+export async function makeNewBlankWindow(): Promise<void> {
+  await runAppleScript(`
+    tell application "Arc"
+	    activate
+    end tell
+    delay(0.5)
```
Comment (danulqua) [inline extensions/arc/src/arc.ts:156]: Why do we need this duplicate with delay?

**Review by danulqua — CHANGES_REQUESTED** [review]

File: extensions/arc/src/arc.ts (hunk 3 above)
Comment (Visual-Studio-Coder) [reply extensions/arc/src/arc.ts:156]: For some reason, it's Diane open a new windows unless you are ON that window, so I had to duplicate it to ensure the user is ON an arc window

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**